src/SpikeCore/SpikeCore/Modules/ModuleBase.cs

```diff
@@ -16,16 +15,24 @@ public abstract class ModuleBase : IMessageHandler<IrcChannelMessageMessage>, IM
         public abstract string Description { get; }
         public abstract string Instructions { get; }
 
+        public virtual List<string> Triggers => new List<string> {Name};
```

Comment:
Would it be a good idea to make this IEnumerable<string> so that implementers have more flexibility in what provides the values?

Reply:
I initially made it a List because I thought I was going to do something with ordering. I wound up just doing an Any and a regex on the other end, so in retrospect I guess we don't need a list.

TL;DR Summary: yeah, sure, this makes sense.